atomic_reactor/tasks/sources.py

```diff
@@ -44,15 +53,17 @@ class SourceBuildTask(plugin_based.PluginBasedTask):
             {"name": "compress", "args": {"load_exported_image": True, "method": "gzip"}},
             {"name": "tag_and_push"},
         ],
-        exit=[
+    )
+
+
+class SourceExitTask(SourceBuildBaseTask):
+    """Source container exit task."""
+
+    plugins_def = plugin_based.PluginsDef(
+        prebuild=[
```

Comment:
Could this be `exit` now?

I think there is some logic in `plugin.py` which makes it so that the `PrebuildPluginsRunner` will refuse to run exit plugins.

Reply:
Done. It's now `exit`.